### ISSUE TITLE
Replace shell commands in wrenhandler with native python

### ISSUE THREAD

**Issue by apengwin** [pywren/pywren#151]
There are a few places in `wrenhandler` where we fork shell processes. It would be better to rewrite these with native python code where possible, which would fix some compatibility issues across platforms. For example:

![screen shot 2017-07-27 at 11 44 35 am](https://user-images.githubusercontent.com/7637700/28686717-0402c792-72c1-11e7-93c6-26055666c4c7.png)

causes Azure function to crash, since Windows doesn't have a `find` command.

**Comment by ericmjonas** [comment]
These are also totally not necessary debug code that we could just abandon.

**Comment by apengwin** [comment]
True, we don't really have to keep the `find`, but there's also a few calls to `rm -Rf` that should be rewritten.
https://github.com/pywren/pywren/blob/master/pywren/wrenhandler.py#L184.

**Comment by ericmjonas** [comment]
Will shutil help with those?
